openshift/ci-operator/build-image/cr.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     elasticsearch:
       nodeCount: 3
       storage:
-        storageClassName: gp2
+        storageClassName: standard
```

Comment:
if you leave this field off it should just use the OCP cluster default

Reply:
lets try... removed the class